xbmc/windowing/wayland/OptionalsReg.cpp

```diff
@@ -11,7 +11,7 @@
 //-----------------------------------------------------------------------------
 // VAAPI
 //-----------------------------------------------------------------------------
-#if defined (HAVE_LIBVA) && defined(HAVE_EGL)
+#if defined(HAVE_LIBVA) && defined(HAS_EGL)
```

Comment:
It's annoying that we have two different conjugations of the same verb. Who is having? Why should the plurality of a libva owner not match the plurality of the egl owner?